src/ctelnet.cpp

```diff
@@ -537,6 +540,23 @@ void cTelnet::handle_socket_signal_hostFound(QHostInfo hostInfo)
     }
 }
 
+void cTelnet::slot_closeFiles()
+{
+    if (loadingReplay) {
+        replayFile.close();
+    }
+}
+
+void cTelnet::slot_openFiles(QString newPath, QString oldPath)
+{
+    if (loadingReplay) {
+        QString replayFileName = replayFile.fileName();
+        replayFileName.replace(oldPath, newPath);
+        replayFile.setFileName(replayFileName);
+        replayFile.open(QIODevice::ReadOnly);
```

Comment:
Does this work? If `loadingReplay` ***is*** true we are reading it a chunk at a time - so will this retain the position we were in the file before the relocation took place.  IIRC different OSs handle the moving of a file that is **open** differently - some OS/file-system drivers are happy to continue supplying data from the old name until it is closed! :stuck_out_tongue:

Given that this is a ***very*** *corner case* (doing a replay at the same time that we try to change where Mudlet stores all it's data) why not just detect it (the presence of a non-null `mudlet::mpToolBarReplay`) and abort the change the save location process?

Reply:
My bad, now it shows...